packages/build-info/src/get-build-info.ts

```diff
@@ -15,23 +15,22 @@ export type Info = {
 export const getBuildInfo = async (opts: ContextOptions) => {
   const context = await getContext(opts)
   let frameworks: any[] = []
-  let buildSystems: BuildSystem[] = []
 
   try {
-    // if the framework or buildSystem detection is crashing we should not crash the build info and package-manager
+    // if the framework  detection is crashing we should not crash the build info and package-manager
     // detection
     frameworks = await listFrameworks({ projectDir: context.projectDir })
-    buildSystems = await detectBuildSystems(context.projectDir, context.rootDir)
   } catch {
     // TODO: build reporting to buildbot see: https://github.com/netlify/pillar-workflow/issues/1001
     // noop
   }
 
-  const info: Info = { frameworks, buildSystems }
+  const info: Info = { frameworks }
 
   // only if we find a root package.json we know this is a javascript workspace
   if (Object.keys(context.rootPackageJson).length > 0) {
     info.packageManager = await detectPackageManager(context.projectDir, context.rootDir)
+    info.buildSystems = await detectBuildSystems(context.projectDir, context.rootDir)
```

Comment:
I would still rather opt for having an additional try catch block (to avoid crashing the package manager detection)

Reply:
The try...catch is not necessary here as I am already handling invalid package.json files gracefully. Is there any other scenario where `detectBuildSystems` may error out?